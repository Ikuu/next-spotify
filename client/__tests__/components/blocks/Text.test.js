import React from 'react';
import { create } from 'react-test-renderer';
import Text from '../../../components/blocks/Text';

test('Text renders', () => {
  const tree = create(<Text>Sample text</Text>).toJSON();
  expect(tree).toMatchSnapshot();
});
