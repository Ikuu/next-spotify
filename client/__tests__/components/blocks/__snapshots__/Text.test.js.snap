// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text renders 1`] = `
<p>
  Sample text
</p>
`;
